syncd/FlexCounter.cpp

```diff
@@ -1590,10 +1590,14 @@ void FlexCounter::flexCounterThreadRunFunction()
         MUTEX_UNLOCK; // explicit unlock
 
         // nothing to collect, wait until notified
+        if (m_runFlexCounterThread) {
```

Comment:
Do you want this changes as well?

Reply:
This is a bug fix we made for a race condition. This is not part of the voq implementation. We observed that sometimes, orchagent will be stuck due to dead lock. This happens in our box. May not be problem with vs or other vendors. I'll remove this fix from this PR.